.github/CODEOWNERS

```diff
@@ -1,21 +1,22 @@
 *       @ovotech/production-eng @ovotech/dex
-/terraform/               @ovotech/production-eng @ovotech/dex
-/clair-scanner/           @ovotech/production-eng @ovotech/dex
-/aws-rotate-keys/         @mwz
-/rac-gcp-deploy/          @ovotech/reads-consumption
-/rac-kmi-deploy/          @ovotech/reads-consumption
-/aws-get-parameters/      @ovotech/jaws
-/tools-install/           @ovotech/orion-sre
-/with-git-deploy-key/     @ovotech/orion-sre
-/versioned-lambda/        @ovotech/orion-migration
-/set-current-environment/ @ovotech/jaws
-/oot-eks/                 @ovotech/orion-ops-tooling
-/clojure/                 @ovotech/orion-ops-tooling
-/ssh-proxy/               @ovotech/orion-security-engineering
-/jaws-journey-deploy      @ovotech/jaws-library-maintainers
-/oot-deploy/              @ovotech/orion-ops-tooling
-/sbt-deploy/              @ovotech/smart-bookings
-/asp-deploy/              @ovotech/asp-team
-/argocd/                  @ovotech/metering-industry-abstraction-cluster-admins
-/comms-commands/          @ovotech/comms
-/ipa-deploy/              @ovotech/ipa
+/terraform/                      @ovotech/production-eng @ovotech/dex
+/clair-scanner/                  @ovotech/production-eng @ovotech/dex
+/aws-rotate-keys/                @mwz
+/rac-gcp-deploy/                 @ovotech/reads-consumption
+/rac-kmi-deploy/                 @ovotech/reads-consumption
+/aws-configure-credentials-oidc/ @ovotech/production-eng
+/aws-get-parameters/             @ovotech/jaws
+/tools-install/                  @ovotech/orion-sre
+/with-git-deploy-key/            @ovotech/orion-sre
+/versioned-lambda/               @ovotech/orion-migration
+/set-current-environment/        @ovotech/jaws
+/oot-eks/                        @ovotech/orion-ops-tooling
+/clojure/                        @ovotech/orion-ops-tooling
+/ssh-proxy/                      @ovotech/orion-security-engineering
+/jaws-journey-deploy             @ovotech/jaws-library-maintainers
+/oot-deploy/                     @ovotech/orion-ops-tooling
+/sbt-deploy/                     @ovotech/smart-bookings
+/asp-deploy/                     @ovotech/asp-team
+/argocd/                         @ovotech/metering-industry-abstraction-cluster-admins
+/comms-commands/                 @ovotech/comms
+/ipa-deploy/                     @ovotech/ipa
```

Comment:
Not sure why Github's complaining about these - that team definitely exists (although it also hasn't changed in this PR so out of scope)

Reply:
Yeah, I noticed that too. Seems to be complaining about some already: https://github.com/ovotech/circleci-orbs/blob/master/.github/CODEOWNERS